rollup.config.js

```diff
@@ -1,10 +1,12 @@
 import resolve from 'rollup-plugin-node-resolve'
 import babel from 'rollup-plugin-babel'
 import minify from 'rollup-plugin-babel-minify'
+import image from 'rollup-plugin-image'
 
 const config = {
   input: 'src/bandicoot.js',
   plugins: [
+    image(),
```

Comment:
this seems necessary for using images in rollup but I couldn't get it to work 😕

Reply:
# The best way to do this
Just embed the SVG right into a react component. You have to modify it a little bit because the react props to `<svg>` are a bit different than the html attributes. But this is by far the easiest way. The example code below doesn't have all the indentation or attributes fixed, but gives you the idea.

```jsx
function BoldIcon() {
  return (
    <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 317.41 317.41" style="enable-background:new 0 0 317.41 317.41;" xml:space="preserve">
<path d="M281.439,158.705c21.727-15.146,35.971-40.316,35.971-68.75c0-46.18-37.57-83.75-83.75-83.75h-40H115H45H0v30h45v245H0v30
	h45h70h78.66h40c46.18,0,83.75-37.57,83.75-83.75C317.41,199.021,303.166,173.851,281.439,158.705z M193.66,36.205
	c29.638,0,53.75,24.112,53.75,53.75s-24.112,53.75-53.75,53.75H115v-107.5H193.66z M115,173.705h78.66
	c29.638,0,53.75,24.112,53.75,53.75s-24.112,53.75-53.75,53.75H115V173.705z"/>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
<g>
</g>
</svg>
  )
}
```

# What I wrote before I realize that:

Seems like rollup-plugin-image creates an image dom node instead of a url? I'm looking through [their docs](https://github.com/rollup/rollup-plugin-image) and they suggest appending it straight to the DOM. I'm more used to [file-loader](https://webpack.js.org/loaders/file-loader/) which keeps it as a separate file.

If it's true that it creates an image element, we might need to use a react ref and appendchild to it in order for it to work with react.

One alternative to this might be https://www.npmjs.com/package/@redsift/rollup-plugin-imagedata. We could take the base64 string and then render a react image with the base64 string as the src:

```jsx
import {base64} from './bold.svg'

<img src={`data:image/svg,${base64}`} />
```
This might be a good solution because it won't require the user of bandicoot to download the icons as separate network requests from the js bundle. That can be a pain to document correctly because people often don't have their webpack configured correctly with a public path and so network requests relative to the